modules/http/src/main/java/org/mule/module/http/internal/listener/DefaultHttpListenerConfig.java

```diff
@@ -320,11 +320,11 @@ public synchronized void stop() throws MuleException
 
     public void stopServer()
     {
-        server.stop();
         if (server instanceof GrizzlyServer)
         {
-            ((GrizzlyServer) server).cleanIdleConnections();
+            ((GrizzlyServer) server).setCleanIdleConnections();
```

Comment:
I would not use setCleanIdleConnections as the name of the function without parmeters. Maybe add a parameter.